packages/secp256k1/secp256k1.0.4.4/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+synopsis: "Elliptic curve library secp256k1 wrapper for Ocaml"
+description: """\
+This library wrap the secp256k1 EC(DSA) library into an OCaml library. At the moment only a subset of functionalities are available:
+- Context: create, clone, destroy, randomize
+- Elliptic curve: public key creation
+- ECDSA: verify, sign, recover
+
+All exchanged data (pubkey, signature, seckey) are represented as hex strings."""
+maintainer: "Davide Gessa <gessadavide@gmail.com>"
+authors: [
+  "Davide Gessa <gessadavide@gmail.com>"
+  "Vincent Bernardoff <vb@luminar.eu.org>"
+  "Yoichi Hirai <i@yoichihirai.com>"
+  "Anton Trunov"
+]
+license: "MIT"
+homepage: "https://github.com/dakk/secp256k1-ml"
+bug-reports: "https://github.com/dakk/secp256k1-ml/issues"
+depends: [
+  "ocaml"
+  "dune" {>= "1.0"}
```

Comment:
```suggestion
  "dune" {>= "1.3"}
```

I think with this change and given that #21428 has been merged CI may succeed this time